code/game/objects/items/stacks/packagewrap.dm

```diff
@@ -195,7 +197,11 @@
 
 /obj/item/delivery/large/attack_hand(mob/user as mob)
 	if(wrapped)
-		wrapped.forceMove(get_turf(src.loc))
+		wrapped.forceMove(get_turf(loc))
```

Comment:
Skips a step upwards in the tree.
The `obj` has to at LEAST be in a `turf` and we already know it's not a turf, so it's just one step less.